modules/service/src/main/scala/lucuma/odb/graphql/ObsAttachmentRoutes.scala

```diff
@@ -97,6 +98,7 @@ object ObsAttachmentRoutes {
             s
               .insertAttachment(user, programId, typeTag, fileName, description, req.body)
               .flatMap(id => Ok(id.toString))
+              .guarantee(req.body.compile.drain)
```

Comment:
Ugh this is a really good question. I'm not sure it is well-specified, it may be implementation specific, and I have to take a look.

Are we currently using Blaze server? Do you think we can switch to Ember?

Reply:
I have no problem switching to Ember, but I think I remember @tpolecat saying something about Ember vs Blaze so we should check with him.

Reply:
Ember was broken last time I tried it but you could give it a try. I would like to get off Blaze.

Reply:
One or more of the Armans have probably fixed it by now.

Reply:
Yeah at this point I'm making any remaining Ember bugs high priority. In the cutting-edge benchmarks its pulled ahead of Blaze.

Reply:
Do you want to switch in a separate PR?

Reply:
Yeah I can work on that. While I do it I will also double-check (and fix if needed) Ember's behavior in this situation.

Reply:
I could do the switch to ember if you want to look at the implementation.

Reply:
Actually, on my attempt I keep getting these

> [ERROR] org.http4s.ember.server.EmberServerBuilderCompanionPlatform - WebSocket connection terminated with exception 
java.util.concurrent.TimeoutException: 60 seconds

So I'm probably missing something...